config.ts

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
```

Comment:
Probably a better name for the file would be `api.ts`. Or even better would move this to `src/services/api/index.ts`